src/hooks/members/useContributorBreakdown.ts

```diff
@@ -22,36 +22,56 @@ const mergeDomains = (
       ...repInfo,
       nativeId: repInfo.domain.nativeId,
       permissions: [],
+      multiSigPermissions: [],
       domainName:
         repInfo.domain.metadata?.name ?? `Domain ${repInfo.domain.nativeId}`,
     };
   }
 
   for (const permDomain of permissions) {
-    const roles = Object.keys(permDomain).reduce((acc, key) => {
-      if (!key.startsWith('role_') || !permDomain[key]) {
-        return acc;
-      }
+    const { roles, multiSigRoles } = Object.keys(permDomain).reduce(
+      (acc, key) => {
+        if (!key.startsWith('role_') || !permDomain[key]) {
+          return acc;
+        }
 
-      const role = Number(key.split('_')[1]) as AvailablePermission;
+        const role = Number(key.split('_')[1]) as AvailablePermission;
```

Comment:
No super happy about extracting the role from the id, but then again that's why we encoded it in the first place, but be able to do stuff like this :)